fractal_server/app/routes/api/v2/submit.py

```diff
@@ -28,13 +31,16 @@
 from ....security import User
 from ._aux_functions import _get_dataset_check_owner
 from ._aux_functions import _get_workflow_check_owner
+from ._aux_functions import check_jobs_list_worker
 
 
 def _encode_as_utc(dt: datetime):
     return dt.replace(tzinfo=timezone.utc).isoformat()
 
 
 router = APIRouter()
+logger = set_logger(__name__)
+LEN_WORKER_JOB_LIST = 50
```

Comment:
I stand corrected: let's make this a fractal-server configuration variable (something like `FRACTAL_API_MAX_JOB_LIST_LENGTH: int = 50`)

Reply:
(this also allow us to set it e.g. to 1 in the tests, without monkeypatching - in case it is relevant)